node-playwright-chrome/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:14-slim
+ARG NODE_VERSION=14
```

Comment:
I think we should change the default to 16 since we use 16 as default in the builds